tests/fixtures/identity.sql

```diff
@@ -381,6 +381,8 @@ CREATE TABLE z (a INT) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARACTER SET=utf8
 CREATE TABLE z (a INT DEFAULT NULL, PRIMARY KEY(a)) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARACTER SET=utf8 COLLATE=utf8_bin COMMENT='x'
 CREATE TABLE z WITH (format = 'parquet') AS SELECT 1
 CREATE TABLE z WITH (format = 'ORC', x = '2') AS SELECT 1
+CREATE TABLE z WITH (table_format = 'iceberg', format = 'parquet') AS SELECT 1
+CREATE TABLE z WITH (table_format = 'iceberg', format = 'ORC', x = '2') AS SELECT 1
```

Comment:
add a test to dialects to test the other way around

Reply:
good call